netlink/link_linux.go

```diff
@@ -426,7 +432,7 @@ func (Netlink) AddOrRemoveStaticArp(mode int, name string, ipaddr net.IP, mac ne
 	state := 0
 	if mode == ADD {
 		req = newRequest(unix.RTM_NEWNEIGH, unix.NLM_F_CREATE|unix.NLM_F_REPLACE|unix.NLM_F_ACK)
-		state = NUD_PERMANENT
+		state = NUD_PROBE
```

Comment:
can we get state as parameter? lets move all pramaeters to linkinfo structure?

Reply:
If we want to do this, we will need to change all the function calls for adding the ARP. Should I do that ? Currently we are not passing link info as a parameter to this function.

Reply:
yep lets do it..its not referenced in that many places..also lets update function name to SetOrRemoveLinkAddress

Reply:
Sure